compaction/compaction.cc

```diff
@@ -1637,8 +1637,8 @@ class scrub_compaction final : public regular_compaction {
         if (!range.is_full()) {
```

Comment:
subject says `full_range`, but code uses `full_scan`.

Reply:
thanks for pointing this out. corrected.